src/redux/store.js

```diff
@@ -1,24 +1,25 @@
 /* eslint-disable import/no-named-as-default-member */
 /* eslint-disable import/no-named-as-default */
 import { configureStore } from '@reduxjs/toolkit';
-import counterReducer from './slices/counter';
+import getProfileReducer from './slices/profile/getProfileSlice.js';
+import updateProfileReducer from './slices/profile/updateProfile';
 import productAddSlice from './slices/product/productAdd';
 import sidebarSlice from './slices/sidebar';
 import categoriesSlice from './slices/product/categories';
 import signupSlice from './slices/user/signup';
 
 const reducers = {
-  counter: counterReducer,
-};
+  profile: getProfileReducer,
+  profileUpdate: updateProfileReducer
+}
 const store = configureStore({
```

Comment:
please follow well other's format don't create you own you are confusing the team